src/main.js

```diff
@@ -108,7 +108,13 @@ if (locale) {
 // In that case, use `en` as the Electron locale.
 
 const electronLocale = (!locale || locale === 'qps-ploc') ? 'en' : locale;
-app.commandLine.appendSwitch('lang', electronLocale);
+
+// For now, don't pass in the locale on macOS due to
+// https://github.com/microsoft/vscode/issues/167543
+
+if (os.platform() !== 'darwin') {
```

Comment:
```suggestion
if (process.platform === 'win32') {
```

There is no WCO for linux, lets make that clear :)